### DISCUSSION TITLE
2022-01-14 Tutorials Read Me Updates

### DISCUSSION THREAD

**Discussion by theo-armour** [konturno/konturno.github.io#11] [Announcements]
@dashdotdotdashdotdot 

https://konturno.github.io/#tutorials/README.md

Added more links. See "Links of Interest" section

**Comment by dashdotdotdashdotdot** [comment]
My first program, I combined two of your lessons. You can "pan the camera
up" by moving all the objects down.  It loads kinda of looking like a flat
design that you can then
rotate and see its 3D

**Comment by dashdotdotdashdotdot** [comment]
Next question, how can I get it to work on my phone or tablet?
